Sources/Private/CoreAnimation/Animations/CALayer+addAnimation.swift

```diff
@@ -205,6 +222,44 @@ extension CALayer {
     return timingFunctions
   }
 
+  /// Creates a `CGPath` for the given `position` keyframes,
+  /// which accounts for `spatialInTangent`s and `spatialOutTangents`
+  private func path<KeyframeValue>(
+    keyframes positionKeyframes: ContiguousArray<Keyframe<KeyframeValue>>,
+    value keyframeValueMapping: (KeyframeValue) throws -> CGPoint) rethrows
+    -> CGPath {
```

Comment:
SwiftFormat doesn't like this open brace for some reason, it won't let me put it on the next line. Never seen this issue before.